src/Home.tsx

```diff
@@ -106,48 +96,26 @@ const Home = ({
 		}
 
 		setLoading(true);
-		api('POST', `/chain`, {
-			"id":"1",
-			"jsonrpc":"2.0",
-			"method":"getstorage",
-			"params":[
-				environment.netmapContract,
-				environment.epochLine,
-			],
-		}).then((res: any) => {
-			if (res['result']) {
-				const epoch_b64: any = atob(res['result']);
-				const epoch_bytes: any = base64ToArrayBuffer(epoch_b64);
-				const dv2: any = new DataView(epoch_bytes.buffer);
-				const epoch: string | number = dv2.getUint32(0, true);
-
-				files.forEach((file: File) => {
-					const formData: any = new FormData();
-					formData.append('file', file);
-					onUploadFile(formData, file.name, epoch, lifetimeData);
-				});
-			} else {
-				setLoading(false);
-				onModal('failed', 'Something went wrong, try again');
-			}
-		}).catch(() => {
-			setLoading(false);
-			onModal('failed', 'Something went wrong, try again');
+		files.forEach((file: File) => {
+			onUploadFile(file);
 		});
     e.preventDefault();
 	};
 
-	const onUploadFile = (formData: any, filename: string | null, epoch: string | number, lifetime: string | number) => {
-		document.cookie = `Bearer=${user.XBearer}; path=/gate/upload; expires=${new Date(Date.now() + 10 * 1000).toUTCString()}`;
-		api('POST', '/gate/upload/', formData, {
-			'X-Attribute-NEOFS-Expiration-Epoch': String(Number(epoch) + Number(lifetime)),
-			'X-Attribute-email': user.XAttributeEmail,
-			'Content-Type': 'multipart/form-data',
+	const onUploadFile = (file: any | null) => {
+		document.cookie = `Bearer=${user.XBearer}; path=/gate/objects; expires=${new Date(Date.now() + 10 * 1000).toUTCString()}`;
+		api('POST', "/gate/objects/", file, {
+			'X-Attributes': JSON.stringify({
+				'FileName': file.name,
+				'Email': user.XAttributeEmail,
+			}),
+			'X-Neofs-Expiration-Duration': lifetimeData,
+			'Content-Type': 'blob',
```

Comment:
Wrong. It depends on the file and you should have it in the browser (at least it was sent correctly previously).